.github/workflows/manual-dispatch.yaml

```diff
@@ -0,0 +1,26 @@
+name: Manual dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      build_file:
+        required: false
+        default: 'Jenkins/Dockerfile'
```

Comment:
This default value points to one Dockerfile (without the suffix). How does one build from a different docker file in manual mode? Our normal manual use-case is to update these docker files together (though we can kick them off one after the other too): Dockerfile.tf-cpu, Dockerfile.tf-gpu, Dockerfile.torch-cpu, Dockerfile.torch-gpu

Reply:
It looks like you can add inputs to manual workflow jobs. See https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

This could be added as a follow up enhancement? Or this could be updated to just build all of them.

Reply:
This already appears to use inputs, so you can specify the file you want to build when manually triggering. That is just the default. See link I provided